### PR TITLE
[241] Parsing and updating tags in the creation

### PR DESCRIPTION
**Pull request by mmiedlarz**
Trello card:
https://trello.com/c/TCTQ4iGW/241-13-as-a-user-i-would-like-to-add-tag-to-creation-during-upload-flow

Just basic parse of tags sent by API

### CODE REVIEW

File: CreatubblesAPIClient/Sources/Model/Creation.swift [hunk 1]
```diff
@@ -189,6 +191,9 @@ open class Creation: NSObject, Identifiable {
 
         contentType = creationEntity.contentType ?? ""
 
+        let entityTags: Array<String> = creationEntity.tags.flatMap { $0.tag }
+        tags = entityTags.isEmpty ? nil : entityTags
```
Comment (sebastianewak) [inline CreatubblesAPIClient/Sources/Model/Creation.swift:195]: Would it make sense to return an empty array here (in the case of `abilities` and `createdAtAgePerCreator` we do provide fallback values)?

**Review by sebastianewak — APPROVED** [review]
Code: +1
Func: n\a

File: CreatubblesAPIClient/Sources/Model/Creation.swift (hunk 1 above)
Comment (mmiedlarz) [reply CreatubblesAPIClient/Sources/Model/Creation.swift:195]: I thought about this, main reason why I left it in this way, was to distinguish situations:
"There were no tags set" vs "There was an empty list of tags" when parsing API

**Review by JakubJankowski — APPROVED** [review]
Looks good! 👍

**Pull request merged**